apps/gallery/index.html

```diff
@@ -57,7 +57,6 @@
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" />
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/bidi-helper.css" />
-    <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script>
```

Comment:
See https://github.com/mozilla-b2g/gaia/tree/master/shared/elements#build-system, I believe for components assets we don't need to comment inside index.html